### DISCUSSION TITLE
Plot Custom Polygon Data

### DISCUSSION THREAD

**Discussion by asimkaymak69** [pyecharts/pyecharts#2355] [Q&A] [answered]
Hello,

I have a custom geopandas file and I can read it with geopandas as a dataframe. I want to plot it with Geo() or Map() objects but I could not found that how can I do that? Do you have any suggestions on this subject?

**Comment by sunhailin-Leo** [comment, marked as answer]
@asimkaymak69 
* I suggest you process it to Python datatypes 😄
